travis.sh

```diff
@@ -2,7 +2,7 @@
 
 set -exo pipefail
 
-VERSION=v2.072.2
+VERSION=v2.076.1
```

Comment:
Didn't someone want us to switch to dmd-cxx branch for the library?

Reply:
We updated `dmd-cxx` at druntime and Phobos to 2.076.1 today :)

Reply:
Yep, I used this until the dmd-cxx branch of the stdlib was updated to check what happens when it tries to build, can just use the dmd-cxx branch now.